extra-cinnamon/nemo/spec

```diff
@@ -1,4 +1,4 @@
-VER=4.6.5
-SRCTBL="https://github.com/linuxmint/nemo/archive/$VER.tar.gz"
-CHKSUM="sha256::35f18063c4e9e0f94997454482553f48296eb97afe13bcbd56e99a87ea8eb78e"
+VER=4.8.0
+SRCS="tbl::https://github.com/linuxmint/nemo/archive/$VER.tar.gz"
+CHKSUMS="sha256::47fec60b3221f2243df1036bb297fbd602947ee3873d05c768e805246a634f52"
 SUBDIR="nemo-$VER"
```

Comment:
I think you can drop all the `SUBDIR=` here. This is a relic from a broken ACBS revision.